doc/INSTALL.md

```diff
@@ -205,7 +205,7 @@ If you need Python 3.x for mako (or get a mako build error):
 If you don't have bitcoind installed locally you'll need to install that
 as well:
 
-    $ brew install berkeley-db4 boost miniupnpc openssl pkg-config libevent libsodium
```

Comment:
Hmm we require `libsodium` and that's the only place it was mentioned, could you instead move it above with `lightningd` deps ?

Reply:
Sure, fixed up.